src/index.js

```diff
@@ -1,5 +1,287 @@
+
+  if (modalOpen) {
+    formContainer.style.pointerEvents = 'none';
+    formContainer.style.transform = 'scale(0)';
+    overlay.style.opacity = 0;
+    modalOpen = false;
+  } else {
+    h2.textContent = 'Update Task';
+    submitInput.value = 'Update';
+    formContainer.style.pointerEvents = 'auto';
+    formContainer.style.transform = 'scale(1)';
+    overlay.style.opacity = 1;
+    modalOpen = true;
+  }
+}
+
+function renderTasks(selectedList) {
+  if (selectedList.tasks.length === 0) {
+    listDisplayContainer.style.background = 'center no-repeat';
+    listDisplayContainer.style.backgroundSize = '35%';
+  } else {
+    listDisplayContainer.style.background = '';
+  }
+
+  selectedList.tasks.forEach((task) => {
+    const taskElement = document.importNode(taskTemplate.content, true);
+    const checkbox = taskElement.querySelector('input');
+    checkbox.id = task.id;
+    checkbox.checked = task.complete;
+    const label = taskElement.querySelector('label');
+    label.htmlFor = task.id;
+
+    const lineBreak = document.createElement('br');
+    label.append(task.name, ', ', task.date, lineBreak, task.description);
+    const editButton = document.createElement('p');
+    editButton.innerHTML = '<i class="far fa-edit"></i>';
+    editButton.classList.add('edit');
+    // eslint-disable-next-line no-use-before-define
+    editButton.addEventListener('click', () => editTask(task, label));
+    const todoTask = taskElement.querySelector('.task');
+    todoTask.append(editButton);
+    tasksContainer.appendChild(taskElement);
+  });
+}
+
+function render() {
+  clearElement(listsContainer);
+  renderLists();
+  const selectedList = lists.find((list) => list.id === selectedListId);
+
+  if (selectedListId === null) {
+    listDisplayContainer.style.display = 'none';
+  } else {
+    listDisplayContainer.style.display = '';
+    listTitleElement.innerHTML = `${selectedList.name}`;
+    renderTaskCount(selectedList);
+    clearElement(tasksContainer);
+    renderTasks(selectedList);
+    colorTasks(selectedList);
+  }
+}
+
+function renderAndSave() {
+  render();
+  localStorage.setItem('task.lists', JSON.stringify(lists));
+  localStorage.setItem('task.selectedListId', selectedListId);
+}
+
+
+function editTask(task, label) {
+  openOrCloseUpdateTaskForm();
+  newTaskInput.value = task.name;
+  newTaskDate.value = task.date;
+  newTaskPriority.value = task.priority;
+  newTaskDescription.value = task.description;
+  newTaskForm.addEventListener('submit', () => {
+    task.name = newTaskInput.value;
+    task.date = newTaskDate.value;
+    task.priority = newTaskPriority.value;
+    task.description = newTaskDescription.value;
+    label.innerHTML = `<span class="checkbox"></span>${task.name}<br>${task.date}<br>${task.description}`;
+    renderAndSave();
+  });
+}
+
+function createList() {
+  return { id: Date.now().toString(), name: newListInput.value, tasks: [] };
+}
+
+function createTask() {
```

Comment:
* [ ] Let us use arrow function and other areas when function keyword was used to define a function